vision_agent/agent/vision_agent.py

```diff
@@ -1,57 +1,39 @@
 import copy
```

Comment:
Looks like you moved all the core functionality of original vision agent to `vision_agent_code.py` and kept just the basic stuff for conversation agent in this file ?

Reply:
Yes, vision_agent_coder should be basically the same as vision_agent was